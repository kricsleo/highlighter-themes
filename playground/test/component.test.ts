import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import Hello from '../src/components/Header.vue'

describe('Hello.vue', () => {
  it('should render', () => {
    const wrapper = mount(Hello)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
