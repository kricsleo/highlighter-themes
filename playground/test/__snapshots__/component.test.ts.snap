// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Hello.vue > should render 1`] = `
"<div text-center=\\"\\"><button i-carbon-sun=\\"\\" dark:i-carbon-moon=\\"\\" text-40=\\"\\" hover:op-80=\\"\\" transition-opacity=\\"\\"></button>
  <p text-20=\\"\\"> You are here. </p>
</div>"
`;
